lib/ui/src/modules/ui/components/stories_panel/index.js

```diff
@@ -84,6 +97,16 @@ StoriesPanel.propTypes = {
   openShortcutsHelp: PropTypes.func,
   name: PropTypes.string,
   url: PropTypes.string,
+  shortcutOptions: PropTypes.objectof(
```

Comment:
typo: `objectOf`

But it's actually not needed here, it should be just `shape`